lib/src/dbs/options.rs

```diff
@@ -19,8 +19,8 @@ pub struct Options {
 	pub db: Option<Arc<String>>,
 	// Connection authentication data
 	pub auth: Arc<Auth>,
-	// How many subqueries have we gone into?
-	pub dive: usize,
+	// Approximately how large is the current call stack?
+	dive: u8,
```

Comment:
`Options` gets `clone`'d a lot. Using a `u8` means less cpu/memory waste for `clone`.